core/listings_test.go

```diff
@@ -104,35 +80,7 @@ func TestListingProtobufAlias(t *testing.T) {
 		}
 		for _, region := range unmarshalledListing.ShippingOptions[0].Regions {
 			if region != pb.CountryCode_ESWATINI {
-				t.Fatal("Error returning pb.CountryCode_ESWATINI")
-			}
-		}
-	}
-	countrycodes2 := [][]pb.CountryCode{
-		{pb.CountryCode(pb.CountryCode_UNITED_KINGDOM), pb.CountryCode(pb.CountryCode_ESWATINI)},
-		{pb.CountryCode(pb.CountryCode_UNITED_KINGDOM), pb.CountryCode(pb.CountryCode_SWAZILAND)},
-
-		{pb.CountryCode(pb.CountryCode_SWAZILAND), pb.CountryCode(pb.CountryCode_ESWATINI)},
-		{pb.CountryCode(pb.CountryCode_ESWATINI), pb.CountryCode(pb.CountryCode_SWAZILAND)},
-		{pb.CountryCode(pb.CountryCode_SWAZILAND)},
-		{pb.CountryCode(pb.CountryCode_ESWATINI)},
-	}
-	for id, cc := range countrycodes2 {
-		listing2 := factory.NewShippingRegionsProtoBufAlias("swaziland_eswatini", cc)
-		marshalled2, _ := proto.Marshal(listing2)
-		unmarshalled2Listing := &pb.Listing{}
-		err := proto.Unmarshal(marshalled2, unmarshalled2Listing)
-		if err != nil {
-			t.Fatal(err)
-		}
-		for _, region := range unmarshalled2Listing.ShippingOptions[0].Regions {
-			if region != pb.CountryCode(pb.CountryCode_SWAZILAND) && region != pb.CountryCode(pb.CountryCode_ESWATINI) {
-				if id == 0 || id == 1 {
-					t.Logf("( %v ) : should fail : SWAZILAND/ESWATINI proto allow_alias ", unmarshalled2Listing.ShippingOptions[0].Regions)
-				}
-				if id != 0 && id != 1 {
-					t.Fatalf("( %v ) : failed : SWAZILAND/ESWATINI proto allow_alias", unmarshalled2Listing.ShippingOptions[0].Regions)
-				}
```

Comment:
Sorry @allen-munsch, your cleverness has out done you. I really didn't understand what was being tested here and felt okay with removing it. Please take a look at the other changes I applied in this commit with regard to the error messages (which still indicate failure but also document how the test is expected to pass).